config/locales/forms/en.yml

```diff
@@ -124,9 +124,12 @@ en:
       continue: Continue
       instructions_text: Use Touch or Face Unlock to access your account with %{app_name}
       instructions_title: Use Touch or Face Unlock to access your account.
-      intro: Enable face or touch unlock as an authentication method on this device.
-        You’ll need this device to use this method in the future. Start by
-        giving your device a nickname.
+      intro_html: 'When you want to access your Login.gov account, you use your device
+        to scan your face or fingerprint. Your device confirms if those scans
+        are a match to the images you already have stored on your device. <p>We
```

Comment:
Since the intro is rendered in a `<p>` tag, this would create a nested paragraph (invalid HTML).

Maybe we could update the view to expect an array of `intro` paragraphs, similar to what we've done in a few other places?

https://github.com/18F/identity-idp/blob/2979497b0f60f0acaa042bbf6655742fd9cc5b5c/app/views/account_reset/request/show.html.erb#L11-L13

Reply:
Good catch, under a strict interpretation the nested paragraphs could be a problem.

I've changed the style of the code to address this.